src/client/java/daos-java/src/main/native/DunsAttribute.pb-c.c

```diff
@@ -643,9 +643,21 @@ static const ProtobufCFieldDescriptor uns__duns_attribute__field_descriptors[7]
     0,NULL,NULL    /* reserved1,reserved2, etc */
   },
   {
-    "on_lustre",
+    "rel_path",
     6,
     PROTOBUF_C_LABEL_NONE,
+    PROTOBUF_C_TYPE_STRING,
+    0,   /* quantifier_offset */
+    offsetof(Uns__DunsAttribute, rel_path),
+    NULL,
+    &protobuf_c_empty_string,
+    0,             /* flags */
+    0,NULL,NULL    /* reserved1,reserved2, etc */
```

Comment:
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)